lib/manager/cargo/extract.ts

```diff
@@ -1,13 +1,21 @@
+import { existsSync, readFileSync } from 'fs';
```

Comment:
Use util/fs

Reply:
Unfortunately, that's not possible - `extractPackageFile` is not an async function and all of `util/fs`'s functions return a promise

Reply:
The function can be made async, check manager api. Promise return is allowed.

So you can use the async methods. 😉